tests/integration/cattletest/core/test_svc_discovery.py

```diff
@@ -3554,3 +3554,56 @@ def test_sidekicks_container_anti_affinity(client, context):
     for instance in instances:
         result[instance.hostId] = True
     assert len(result) == 3
```

Comment:
Would be nice to have a test validating the case when requested ip address is already allocated to existing service/container.